### PR TITLE
feat: New sample for Cloud Compute OS Config Policy

### PR DESCRIPTION
**Pull request by rajasnathak**
## Description

Prepared an example for os_config_policy for inclusion in C.G.C. documentation.

Intending to use this snippet on the following page: https://cloud.google.com/compute/docs/os-configuration-management/create-os-policy-assignment

Note: If you are not associated with Google, open an issue for discussion before submitting a pull request.

## Checklist
  
**Readiness**

- [x] Yes, **merge** this PR after it is approved
- [ ] No, don't **merge** this PR after it is approved

**Style**
  
- [x] My sample follows the rules described for Terraform in the [Effective Samples style guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] My sample follows the other requirements and best practices in the [Contributing 
guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#other-requirements-and-best-practices)
  
**Testing**
  
- [x] I have performed tests described in the [Contributing guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md):
  
   - [x] **[Tests](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#set-up-the-test-environment)** pass: `terraform apply`
   - [x] **[Lint](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#linting-and-formatting)** pass: `terraform fmt` check
  
**Intended location**
  
- [x] Yes, this sample will be (or already is) included on cloud.google.com
      Location(s): https://cloud.google.com/compute/docs/os-configuration-management/create-os-policy-assignment

- [ ] No, this sample won't be included on cloud.google.com
      Reason:
  
**API enablement**

- [x] If the sample needs an API enabled to pass testing, I have added the service to the [Test setup file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/test/setup/main.tf)

**Review**

- [ ] If this sample adds a new directory, I have added codeowners to the [CODEOWNERS file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/.github/CODEOWNERS)

### CODE REVIEW

File: os_config_policy/main.tf [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+# Sample OS policy based on example - https://cloud.google.com/compute/docs/os-configuration-management/working-with-os-policies#example-2
```
Comment (msampathkumar) [inline os_config_policy/main.tf:18]: This URL is different from what is mentioned in the PR title.

https://cloud.google.com/compute/docs/os-configuration-management/create-os-policy-assignment

**Comment by rajasnathak** [comment]
Seems like it failed due to an unrelated issue

**Review by msampathkumar — CHANGES_REQUESTED** [review]
Please update the sample location. Move it to `compute` folder.

**Review by msampathkumar — APPROVED** [review]
LGTM

**Pull request merged**